test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -633,3 +634,13 @@ func getCustomVolumeFactory(pvTemplate *v1.PersistentVolume) func(id int) *v1.Pe
 		return pv
 	}
 }
+
+func isRetryableAPIError(err error) bool {
+	// These errors may indicate a transient error that we can retry in tests.
+	if apierrors.IsInternalError(err) || apierrors.IsTimeout(err) || apierrors.IsServerTimeout(err) {
+		return true
+	}
+	// If the error sends the Retry-After header, we respect it as an explicit confirmation we should retry.
+	_, shouldRetry := apierrors.SuggestsClientDelay(err)
```

Comment:
per the comment of `SuggestsClientDelay(xx)`, I don't think this method tell us it should be retry or not.
```
// ...It does not address whether the error *should* be retried, since some errors (like a 3xx) may
// request delay without retry.